src/Microsoft.Health.Dicom.Api/Features/Context/DicomRequestContextMiddleware.cs

```diff
@@ -51,8 +53,12 @@ public async Task Invoke(HttpContext context, IDicomRequestContextAccessor dicom
 
             dicomRequestContextAccessor.RequestContext = dicomRequestContext;
 
-            // Call the next delegate/middleware in the pipeline
-            await _next(context);
+            // Adding operation id for this request thread to the log context.
+            using (LogContext.PushProperty(OperationIdName, System.Diagnostics.Activity.Current?.RootId))
```

Comment:
I don't see my previous comments, can we sync on this.

Reply:
I have reactivated the previous conversation and replied to that. Feel free to ping me whenever is feasible for you, and we can connect.